daemon/api_test.go

```diff
@@ -108,10 +110,15 @@ func (s *apiSuite) SetUpTest(c *check.C) {
 		configs: map[string]string{},
 	}
 	s.d = nil
+	// Disable real security backends for all API tests
+	s.restoreBackends = ifacestate.MockSecurityBackendsForSnap(
```

Comment:
How about making that simply `MockSecurityBackends(nil)`?  I don't think any call site is using per-snap backends, which means all of that boilerplate is just unnecessary trouble.